src/deck/drivers/src/ledring12.c

```diff
@@ -1146,6 +1146,12 @@ PARAM_GROUP_START(ring)
  * | 10 | Boat lights         | \n
  * | 11 | Alert               | \n
  * | 12 | Gravity             | \n
+ * | 13 | Virtual Memory      | \n
+ * | 14 | Fade color          | \n
+ * | 15 | Communication Signal Strength                | \n
+ * | 16 | Status Localization Service| \n
+ * | 17 | LED timing from memory       | \n
+ * | 18 | Lighthouse  Positioning      | \n
```

Comment:
This breaks the formatting somewhat, the rest of the table has the `| \n` aligned, and this breaks this, to keep it pretty this could be:

```
[...]
 * | 8  | Factory test                   | \n
 * | 9  | Battery status                 | \n
 * | 10 | Boat lights                    | \n
 * | 11 | Alert                          | \n
 * | 12 | Gravity                        | \n
 * | 13 | Virtual Memory                 | \n
 * | 14 | Fade color                     | \n
 * | 15 | Communication Signal Strength  | \n
 * | 16 | Status Localization Service    | \n
 * | 17 | LED timing from memory         | \n
 * | 18 | Lighthouse  Positioning        | \n
```
?

Reply:
Jup good point. It looks fine on the rendering but readability in code is important too